scripts/make.js

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env node
+
+// /scripts/make
+// ShellJS 'makefile' to run various tasks
+
+import Process from 'node:process'
+
+import 'shelljs/make.js'
+import Shell from 'shelljs'
+import Chalk from 'chalk'
```

Comment:
What does this gain us over `npm run ...` scripts? I see that there's logging, but are there any other benefits?

One thing I like about the original `make` command is that it understood dependencies, and so it would automatically skip recompiling things when the source hadn't changed, but I don't think we're getting that here (?)

(Also, this thing should compile fast enough that it wouldn't really matter here...)

Reply:
You are right, it is a bit of a complication. The reason I introduced it was because I was appending stuff to the built files and it was getting a bit messy - this is largely fixed with the usage of esbuild.

Reverted to good old npm scripts!

Reply:
> (Also, this thing should compile fast enough that it wouldn't really matter here...)

ESBuild is indeed blazingly fast - it takes just 2 ms to build the library! 🎉